src/current/v25.4/transaction-diagnostics.md

```diff
@@ -0,0 +1,166 @@
+- `created`: Returns `true` if the request is successfully created, or `false` if the specified fingerprint ID does not exist.
+
+## Troubleshooting example
+
+To troubleshoot with a transaction diagnostics bundle, follow these steps:
+
+1. [Identify the transaction fingerprint ID](#step-1-identify-the-transaction-fingerprint-id) to generate a bundle for.
+1. [Create a bundle request](#step-2-create-a-bundle-request) for that fingerprint ID.
+1. [Download the bundle](#step-3-download-the-bundle) from [DB Console]({% link {{ page.version.version }}/ui-overview.md %}).
+1. [Inspect the bundle](#step-4-inspect-the-bundle).
+
+### Setup
+
+For this example, set the [`application_name`]({% link {{ page.version.version }}/map-sql-activity-to-app.md %}) to enable filtering:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+SET application_name = 'cockroachdb_test';
+~~~
+
+Run the following explicit transaction, which sleeps for 10 seconds:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+BEGIN; SELECT pg_sleep(10), 'cockroachdb_test' ; COMMIT;
+~~~
+
+### Step 1. Identify the transaction fingerprint ID
+
+Identify the transaction fingerprint ID of the example transaction by navigating to the [**SQL Activity**]({% link {{ page.version.version }}/ui-overview.md %}#sql-activity) page in the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}). On the [**Transactions**]({% link {{ page.version.version }}/ui-transactions-page.md %}) tab, search the Top `500` by `Transaction Time` in the `Past Hour` and click **Apply** to list transactions. Filter the transactions by **Application Name**:  `cockroachdb_test`.
+
+In the **Transactions** column, click the transaction fingerprint `SELECT pg_sleep(_), _` to open the [**Transaction Details**]({% link {{ page.version.version }}/ui-transactions-page.md %}#transaction-details-page) page for that fingerprint.
+
+<img src="{{ 'images/v25.4/transaction-diagnostics-1.png' | relative_url }}" alt="Transactions tab" style="border:0px solid #eee;max-width:100%" />
+
+From the **Transaction Details** page, copy the hexadecimal **Fingerprint ID** for this transaction, `afdd4059a899442e`.
+
+<img src="{{ 'images/v25.4/transaction-diagnostics-2.png' | relative_url }}" alt="Transactions Details page" style="border:0px solid #eee;max-width:100%" />
+
+Note the decimal equivalent of the fingerprint ID in the browser's address bar. In this case, the URL may look like `https://127.0.0.1:29001/#/transaction/12672355680315327534?appNames=cockroachdb_test`. The decimal value of the fingerprint is `12672355680315327534`.
+
+### Step 2. Create a bundle request
+
+Create a transaction diagnostics request to be fulfilled the next time the relevant transaction is executed:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+SELECT * FROM crdb_internal.request_transaction_bundle('afdd4059a899442e', 0, '0', '0', false);
+~~~
+
+where
```

Comment:
"where" is a bit informal and also unclear since the previous punctuation (colon followed by a thing) suggests the previous sentence was complete. 

Suggest making this a new sentence like "The parameters to the call to `crdb_internal.request_transaction_bundle()` are:" or something like that